src/nginxconfig/generators/conf/website.conf.js

```diff
@@ -60,13 +60,21 @@ const httpsListen = domain => {
     const config = [];
 
     // HTTPS
-    config.push(['listen', `${domain.server.listenIpv4.computed === '*' ? '' : `${domain.server.listenIpv4.computed}:`}443 ssl${domain.https.http2.computed ? ' http2' : ''}`]);
-
+    config.push(['listen', 
+        `${domain.server.listenIpv4.computed === '*' ? '' : `${domain.server.listenIpv4.computed}:`}443 ssl` 
+        + `${domain.https.http2.computed ? ' http2' : '' }`
+        + `${domain.https.http3.computed ? ' http3' : ''}`
+        + `${domain.https.portReuse.computed ? ' reuseport' : ''}`,
+    ]);
+    
     // v6
     if (domain.server.listenIpv6.computed)
         config.push(['listen',
-            `[${domain.server.listenIpv6.computed}]:443 ssl${domain.https.http2.computed ? ' http2' : ''}`]);
-
+            `[${domain.server.listenIpv6.computed}]:443 ssl`
+            + `${domain.https.http2.computed ? ' http2' : '' }`
+            + `${domain.https.http3.computed ? ' http3' : ''}`
+            + `${domain.https.portReuse.computed ? ' reuseport' : ''}`,
+        ]);
```

Comment:
Doesn't http3 need to be in a separate listen directive as it uses udp rather than tcp?

Reply:
yes indeed my bad

Reply:
Is there a way to add comments to the lines in the config. I would like to add UDP listener / TCP listener

Reply:
Yeah, you can just push a comment:

https://github.com/digitalocean/nginxconfig.io/blob/eb25fc5eeec4ff103dcaa9c164b9d0d3cf873430/src/nginxconfig/generators/conf/website.conf.js#L47